e2e/libs/node/aws.py

```diff
@@ -5,7 +5,7 @@
 
 class EC2:
 
-    _ec2_instance = boto3.resource('ec2')
+    _ec2_instance = boto3.resource('ec2',region_name='us-west-1')
```

Comment:
`aws` and `cloud_provider` related code could be removed later if `litmus chaos` can be integrated into negative test. FYI